tfjs-core/src/tensor.ts

```diff
@@ -236,6 +236,12 @@ export declare namespace Tensor {}
  * A `tf.Tensor` object represents an immutable, multidimensional array of
  * numbers that has a shape and a data type.
  *
+ * For performance reasons, functions that create tensors do not necessarily
+ * perform a copy of the data passed to them (e.g. if the data is passed as a
+ * `Float32Array`), and changes to the data will change the tensor. This is not
+ * a feature and is not supported. To avoid this behavior, use the tensor before
+ * changing the input data or create a copy with `copy = tf.add(yourTensor, 0)`.
```

Comment:
will `tf.clone(x)` be a better suggestion? since it does not involve computation. 
But I am not sure if the result would also suffer the mutability issue.

Reply:
`tf.clone(x)` does not work. I just tested it, and it creates a reference to the original tensor's data instead of performing a copy.